sdk/core/Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
-name          = "azure_sdk_core"
-version       = "0.43.7"
-description   = "Rust wrappers around Microsoft Azure REST APIs - Core crate"
-readme        = "README.md"
-authors       = ["Microsoft Corp."]
-license       = "Apache-2.0"
-repository    = "https://github.com/azure/azure-sdk-for-rust"
-documentation = "http://mindflavor.github.io/AzureSDKForRust/azure_sdk_for_rust/index.html"
-homepage      = "https://github.com/azure/azure-sdk-for-rust"
-keywords      = ["sdk", "azure", "rest", "iot", "cloud"]
-categories    = ["api-bindings"]
-edition       = "2018"
+name           = "azure_core"
+version        = "0.1.0"
```

Comment:
What tool is being used to align everything? Aligning stuff actually makes it more difficult to search and replace for the dependencies.

Reply:
I've been doing it by hand since they were formatted this way before. I'm all for not formatting them this way.